sentry-config.cmake.in

```diff
@@ -1,34 +1,27 @@
 @PACKAGE_INIT@
-include(CMakeFindDependencyMacro)
 
 set(SENTRY_BACKEND @SENTRY_BACKEND@)
 set(SENTRY_TRANSPORT @SENTRY_TRANSPORT@)
 set(SENTRY_BUILD_SHARED_LIBS @SENTRY_BUILD_SHARED_LIBS@)
 set(SENTRY_LINK_PTHREAD @SENTRY_LINK_PTHREAD@)
 
-if(SENTRY_BACKEND STREQUAL "crashpad")
-	include("${CMAKE_CURRENT_LIST_DIR}/sentry_crashpad-targets.cmake")
-	if(NOT MSVC AND NOT SENTRY_BUILD_SHARED_LIBS)
-		find_dependency(ZLIB REQUIRED)
-	endif()
-endif()
-
-if(SENTRY_BACKEND STREQUAL "breakpad" AND NOT SENTRY_BUILD_SHARED_LIBS)
-	set(SENTRY_BREAKPAD_SYSTEM @SENTRY_BREAKPAD_SYSTEM@)
-	if(SENTRY_BREAKPAD_SYSTEM)
-		find_dependency(PkgConfig REQUIRED)
+if(NOT SENTRY_BUILD_SHARED_LIBS)
+	include(CMakeFindDependencyMacro)
+	if(SENTRY_BACKEND STREQUAL "breakpad" AND "@SENTRY_BREAKPAD_SYSTEM@")
+		find_dependency(PkgConfig)
 		pkg_check_modules(BREAKPAD REQUIRED IMPORTED_TARGET breakpad-client)
 	endif()
+	if("@SENTRY_TRANSPORT_CURL@")
+		find_dependency(CURL COMPONENTS AsynchDNS)
+	endif()
+	if("@SENTRY_TRANSPORT_COMPRESSION@" AND "@CRASHPAD_ZLIB_SYSTEM@")
+		find_dependency(ZLIB)
+	endif()
+	if(SENTRY_LINK_PTHREAD)
+		set(THREADS_PREFER_PTHREAD_FLAG ON)
+		find_dependency(Threads REQUIRED)
```

Comment:
Is there a reason why you left this `REQUIRED` here?

Reply:
No, just an oversight. :facepalm: